CSS/components/card.css

```diff
@@ -0,0 +1,64 @@
+.card{
+    background-color: var(--paragraph-color);
+    max-width: 19rem;
+    height: fit-content;
+    padding-bottom: 0.5rem;
+}
+.footer{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+.icons{
+    border-radius: 0.5rem;
+    padding: 0.3rem;
+}
+.icons:hover{
+    cursor: pointer;
+    background-color: var(--active-hover-color);
+    color: white;
+}
+.small-text{
+    font-size: 0.8rem;
+}
+.hearder-text-primary{
+    color: var(--button-text-color);
+}
+.padding-left-right-sm{
```

Comment:
You can rename it as pd-horizontal less words to denote the meaning else this one is also fine to go with.

Reply:
Ya renamed the class name to pd-horizontal